cmd/sonobuoy/app/operations/logs.go

```diff
@@ -17,11 +17,66 @@ limitations under the License.
 package operations
 
 import (
-	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/heptio/sonobuoy/pkg/config"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	podLogSeparator = strings.Repeat("-", 79)
 )
 
-// Logs gathers the logs for the containers in the sonobuoy namespace and prints them out in a structured format
-func GetLogs( /*opts?*/ ) error {
-	// This should be straight forwards
-	return errors.New("not implemented")
+// Logs gathers the logs for the containers in the sonobuoy namespace and prints them
+
+// GetLogs streams logs from the sonobuoy pod by default to stdout.
+func GetLogs(cfg *config.Config, follow bool) error {
```

Comment:
could `config.Config` be a bit narrower? Seems like a big thunk to pass in to a single log function

Reply:
yeah it is, i only need the namespace from the config so no reason to pass it in at all